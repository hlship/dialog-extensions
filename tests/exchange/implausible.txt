talk to malory
good
precautions
safe house
sell
pass
z
