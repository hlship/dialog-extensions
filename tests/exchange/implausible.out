

Another miserable night in the London fog and rain. You have the package, and
Malory is your contact... but can he be trusted?

Foggy Alley
You would be hard pressed to find a more cliche location for the drop off than
this deserted alley. Fog billows in from somewhere, and a lone lamppost sheds
minimal light. The sounds of bustling London are distant.

Malory, your contact, stands just outside the weak cone of light from the
lamppost.

> talk to malory
"Malory" is all you say. We're taught to keep these exchanges brief.

"Well," asks Malory, "do you have the package?" Always a bit too direct, this
one, everything's so urgent, no sense of panache.

You could say all in good time.

> good
"All in good time, Malory. There's a protocol to these things."

"Home Office was very clear," he responds, "there's a timetable to keep."

You could ask if all precautions have been made.

> precautions
"Fine, then. On to business," you begin, "have all proper precautions been
taken?"

Malory gives you a hard stare. "Damn it, man. This is important--what are you
playing at?"

You could ask whether the safe house is ready or ask for the pass phrase.

> safe house
"The safe house, is it ready?" you inquire.

Malory scoffs: "Yes, there's a safe house. No, I won't reveal its location. This
is not a training exercise, why are you being so obtuse?"

A moment passes.

Malory asks "What the devil are you up to? This whole exchange should be cut and
dried." He searches out your face in the gloom, looking for something.

You could ask for the pass phrase or comment on the weather.

> sell
You try a joke: "Planning on selling this to the highest bidder? Might be your
best shot out of the game."

Malory is taken aback. "If that's an attempt at humor, now is not the time or
place. We'll be having words about this back at the den."

Perhaps this was not the best gambit. Malory has friends among the Oxford crowd,
so the consequences of this unseemly exchange could be quite dire.

A moment passes.

"Perhaps field work has grown too taxing for you," he says, "we may need to have
a post-mortem back at the den." That sounds ominous.

You could ask for the pass phrase or comment on the weather.

> pass
"I'm certainly not handing over something this valuable without the proper pass
phrase," you announce.

Malory's eyes flash left and right, and he leans in close. He whispers the
correct pass phrase almost inaudibly. You're somewhat relieved.

You could comment on the weather.

> z
A moment slips away.

Are you satisfied? Hard to say in this business. People have secrets, wheels
within wheels. Still, it's time to hand him the package and hope for the best.

You could comment on the weather.

> 
